### ISSUE TITLE
Failed to cleanup datastore after 10 seconds

### ISSUE THREAD

**Issue by zsz00** [JuliaParallel/DTables.jl#10]
I try the example of  https://juliaparallel.org/DTables.jl/dev/dtable/#Table-operations
run is ok, but exit() repl very slow!

```
julia> exit()
┌ Warning: Failed to cleanup datastore after 10 seconds
│ Forcibly evicting all entries
└ @ MemPool ~/.julia/packages/MemPool/k1XDl/src/MemPool.jl:144

```

```
julia> versioninfo()
Julia Version 1.8.0-rc1
Commit 6368fdc656 (2022-05-27 18:33 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: 8 × 11th Gen Intel(R) Core(TM) i5-11300H @ 3.10GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-13.0.1 (ORCJIT, tigerlake)
  Threads: 1 on 8 virtual cores

(@v1.8) pkg> st DTables
Status `~/.julia/environments/v1.8/Project.toml`
  [20c56dc6] DTables v0.1.0
```

**Comment by jpsamaroo** [comment]
Yeah this is a MemPool bug. I'll switch this to an `@debug` and default it to a small delay. Will update this issue soon.

**Comment by jpsamaroo** [comment]
Fixed with MemPool `master`, I'm tagging 0.4.2 with this change.
